jupyterlab/tests/test_announcements.py

```diff
@@ -62,7 +62,7 @@ async def test_CheckForUpdateHandler_get_pypi_success(mock_client, labserverapp,
     payload = json.loads(response.body)
     assert payload["notification"]["message"] == message
     assert payload["notification"]["link"] == [
-        "Open changelog",
+        "Changelog",
```

Comment:
Tests are failing, it looks like this needs to be aligned with the others:

```suggestion
        "Read more…",
```